app/models/form1095_b.rb

```diff
@@ -0,0 +1,137 @@
+        "topmostSubform[0].Page1[0].Part1Contents[0].Line1[0].f1_01[0]": data['first_name'],
+        "topmostSubform[0].Page1[0].Part1Contents[0].Line1[0].f1_02[0]": data['middle_name'],
+        "topmostSubform[0].Page1[0].Part1Contents[0].Line1[0].f1_03[0]": data['last_name'],
+        "topmostSubform[0].Page1[0].Part1Contents[0].f1_04[0]": data['ssn'] || '',
+        "topmostSubform[0].Page1[0].Part1Contents[0].f1_05[0]": data['ssn'] ? '' : data['birth_date'],
+        "topmostSubform[0].Page1[0].Part1Contents[0].f1_06[0]": data['address'],
+        "topmostSubform[0].Page1[0].Part1Contents[0].f1_07[0]": data['city'],
+        "topmostSubform[0].Page1[0].Part1Contents[0].f1_08[0]": data['state'] || data['province'],
+        "topmostSubform[0].Page1[0].Part1Contents[0].f1_09[0]": country_and_zip,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].f1_25[0]": data['first_name'],
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].f1_26[0]": middle_initial,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].f1_27[0]": data['last_name'],
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].f1_28[0]": data['ssn'] || '',
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].f1_29[0]": data['ssn'] ? '' : data['birth_date'],
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_01[0]": data['coverage_months'][0] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_02[0]": data['coverage_months'][1] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_03[0]": data['coverage_months'][2] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_04[0]": data['coverage_months'][3] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_05[0]": data['coverage_months'][4] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_06[0]": data['coverage_months'][5] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_07[0]": data['coverage_months'][6] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_08[0]": data['coverage_months'][7] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_09[0]": data['coverage_months'][8] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_10[0]": data['coverage_months'][9] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_11[0]": data['coverage_months'][10] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_12[0]": data['coverage_months'][11] && 1,
+        "topmostSubform[0].Page1[0].Table1_Part4[0].Row23[0].c1_13[0]": data['coverage_months'][12] && 1
+      },
+      flatten: true
+    )
+    pdf = temp_file.read
+
+    temp_file.close
+    temp_file.unlink
+
+    pdf
+  rescue PdfForms::PdftkError => e
+    # in case theres other errors generating the PDF
+    Rails.logger.error e
+    raise e
```

Comment:
Are you trying to re-raise the error here? If so you will just need to do `raise`
```suggestion
    raise
```